### PR TITLE
fix: null pointer when HWF not entered

### PR DESCRIPTION
**Pull request by SebFlippence**
### JIRA link (if applicable) ###

[NFDIV-680](https://tools.hmcts.net/jira/browse/NFDIV-680) Payment - State Changes

### Change description ###

Fixes a null pointer exception when HWF is not set.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/divorce/citizen/event/CitizenSubmitApplication.java [hunk 1]
```diff
@@ -77,7 +77,7 @@ public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuild
         }
 
         State state;
-        if (caseDataCopy.getHelpWithFeesAppliedForFees().toBoolean()) {
+        if (caseDataCopy.getHelpWithFeesAppliedForFees() != null && caseDataCopy.getHelpWithFeesAppliedForFees().toBoolean()) {
```
Comment (nitinprabhu) [inline src/main/java/uk/gov/hmcts/divorce/citizen/event/CitizenSubmitApplication.java:80]: can front end set this to No by default ?

**Review by velimd — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/divorce/citizen/event/CitizenSubmitApplication.java (hunk 1 above)
Comment (SebFlippence) [reply src/main/java/uk/gov/hmcts/divorce/citizen/event/CitizenSubmitApplication.java:80]: `helpWithFeesNeedHelp` always gets set to `Yes`/`No` while `helpWithFeesAppliedForFees` only gets set once the first question is `Yes`. Perhaps we should be either checking if `helpWithFeesNeedHelp` 
 is true or if `helpWithFeesReferenceNumber` is not null and not empty

Comment (velimd) [reply src/main/java/uk/gov/hmcts/divorce/citizen/event/CitizenSubmitApplication.java:80]:
```suggestion
        if (caseDataCopy.getHelpWithFeesNeedHelp().toBoolean() && caseDataCopy.getHelpWithFeesAppliedForFees().toBoolean()) {
```

Comment (SebFlippence) [reply src/main/java/uk/gov/hmcts/divorce/citizen/event/CitizenSubmitApplication.java]: I've switched this to use `helpWithFeesNeedHelp` instead which will always be set.

**Review by nitinprabhu — APPROVED** [review]

**Review by velimd — APPROVED** [review]

**Pull request merged**